shared/actions/wallets.js

```diff
@@ -158,6 +194,13 @@ function* walletsSaga(): Saga.SagaGenerator<any, any> {
     navigateToAccount
   )
   yield Saga.safeTakeEveryPure(WalletsGen.accountsReceived, maybeSelectDefaultAccount)
+  yield Saga.actionToPromise(WalletsGen.setBuildingAmount, buildPayment)
```

Comment:
You should have an array of actions not a bunch of single ones. Do [], buildPayment